pillar.example

```diff
@@ -105,7 +105,16 @@ nginx:
       - upstream netdata:
           - server: 127.0.0.1:19999
           - keepalive: 64
-
+  
+  streams:
+    example_stream:
+      - upstream some_upstream:
+        - server: '10.0.0.1:8000'
+        - server: '10.0.0.2:8000'
+      - server:
+        - listen: 7000
+        - proxy_pass: some_upstream
+        
```

Comment:
There are whitespaces after the closing bracket, that tests are showing. Can you remove them?